tests/test_openapi.py

```diff
@@ -1,21 +1,20 @@
 import json
 import pkg_resources
 
-import swagger_spec_validator.util
-import yaml
-from swagger_spec_validator.common import SwaggerValidationError
+from openapi_spec_validator import validate_spec
+from openapi_spec_validator.exceptions import OpenAPIValidationError
+from openapi_spec_validator.readers import read_from_filename
```

Comment:
We generally try to stick to import just the modules for clarity while reading code. Something like this
```suggestion
import openapi_spec_validator
from openapi_spec_validator import exceptions as specs_exceptions
from openapi_spec_validator import readers as specs_readers
```